src/app/components/ImageCard/ImageCard.tsx

```diff
@@ -0,0 +1,42 @@
+import Image from 'next/image';
+
+interface ImageCardProps {
+  imageUrl: string;
+  imageAlt: string;
+  children: React.ReactNode;
+  imageFirst?: boolean;
+  imageWidth?: number;
+  imageHeight?: number;
+}
+
+export function ImageCard({
+  imageUrl,
+  imageAlt,
+  children,
+  imageFirst = true,
+  imageWidth = 0,
+  imageHeight = 0,
+}: ImageCardProps) {
+  return (
+    <div className="image-card grid grid-cols-1 justify-items-center gap-4 xl:grid-cols-[1fr_2fr] xl:justify-items-start xl:gap-14">
+      <div
+        className={`image-box h-[100%] w-[30rem] overflow-hidden ${
```

Comment:
We can swap `h-[100%]` for Tailwind's `h-full` since they're equivalent.
```suggestion
        className={`image-box h-full w-[30rem] overflow-hidden ${
```

Reply:
Done